uber/templates/signups/shifts.html

```diff
@@ -83,7 +83,7 @@
                     end : "{{ job.end_time_local|datetime("%Y-%m-%dT%H:%M:%S") }}",
                     id : "{{ job.id }}",
                     description: "{{ job.description }}",
-                    extra15: "{{ job.extra15 }}",
+                    extra15: {{ job.extra15|string|lower }},
```

Comment:
This was rendering the booleans as a string: `extra15: "False",` and `extra15: "True",`.  When we really want those to render as a javascript boolean literal: `extra15: false,` and `extra15: true,`.

Reply:
Ahh, make sense! Nice catch!